Panic during plan

Not sure if this could be user error? This is on v0.3.22 ~~v0.3.23~~ and we've recently updated from v0.2.0. I've moved back to v0.3.10 and don't see the same error there. I've confirmed the error on all versions from v0.3.18-0.3.23.

```
Lock provider has been created successfully
Using GitHub.
GitHub context parsed successfully
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x14e25e8]

goroutine 1 [running]:
github.com/diggerhq/digger/cli/pkg/digger.UpdateStatusComment({0xc000590c00?, 0xe, 0x35ff0a0?}, 0x0?, {0x25947d0, 0xc000809830}, 0xffffffffffffffff?)
	/home/runner/work/digger/digger/cli/pkg/digger/digger.go:169 +0xe8
main.gitHubCI({0x25876c8, 0xc00007a0a0}, {0x2586cd8, 0xc000b10990}, {0x2586d08, 0xc000b18cf0}, {0x2574440?, 0xc000b22000})
	/home/runner/work/digger/digger/cli/cmd/digger/main.go:128 +0xcd1
main.glob..func2(0xc0000aa200?, {0x1f47aa6?, 0x4?, 0x1f47aaa?})
	/home/runner/work/digger/digger/cli/cmd/digger/default.go:23 +0x205
github.com/spf13/cobra.(*Command).execute(0x35d1980, {0x3639ca0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xaa3
github.com/spf13/cobra.(*Command).ExecuteC(0x35d2220)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
main.main()
	/home/runner/work/digger/digger/cli/cmd/digger/main.go:815 +0xf4
Error: Process completed with exit code 2.
```

Thanks, @motatoes. Here's the output with v0.3.25:

```
Error while fetching user teams for CI service: failed to list github teams: GET https://api.github.com/orgs/identiv/teams: 403 Resource not accessible by integration []
WARNING: teams failed to be fetched, passing an empty list instead for access policy checks
DEBUG: passing the following input policy: map[action:digger plan approvals:[] organisation:identiv planPolicyViolations:[] project:**snip** teams:[] user:macbutch] ||| text: 
package digger
default allow = true
allow = (count(input.planPolicyViolations) == 0)
Lock identiv/**snip**
Lock result: false
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x14e1f55]
goroutine 1 [running]:
github.com/diggerhq/digger/cli/pkg/digger.RunJobs({0xc000bcca90, 0x1, 0x2?}, {0x2594950, 0xc000624120}, {0x25746e0, 0xc000624120}, {0x2587848, 0xc00007c530}, {0x257fe68, ...}, ...)
	/home/runner/work/digger/digger/cli/pkg/digger/digger.go:118 +0xab5
main.gitHubCI({0x2587848, 0xc00007c530}, {0x2586e58, 0xc000b53230}, {0x2586e88, 0xc000b5ebd0}, {0x25745c0?, 0xc000607320})
	/home/runner/work/digger/digger/cli/cmd/digger/main.go:327 +0x2419
main.glob..func2(0xc0000c1400?, {0x1f47b06?, 0x4?, 0x1f47b0a?})
	/home/runner/work/digger/digger/cli/cmd/digger/default.go:23 +0x205
github.com/spf13/cobra.(*Command).execute(0x35d2980, {0x363aca0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xaa3
github.com/spf13/cobra.(*Command).ExecuteC(0x35d3220)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
main.main()
	/home/runner/work/digger/digger/cli/cmd/digger/main.go:819 +0xf4
```

I've redacted the projected name. Let me know if you need it, I'll share some other way.

--- a/cli/pkg/digger/digger.go
+++ b/cli/pkg/digger/digger.go
@@ -482,7 +482,7 @@ func run(command string, job orchestrator.Job, policyChecker policy.Checker, org
 		msg := fmt.Sprintf("Command '%s' is not supported", command)
 		return nil, msg, fmt.Errorf(msg)
 	}
-	return nil, "", nil
+	return &execution.DiggerExecutorResult{}, "", nil
 }
 
 func retrievePlanBeforeApply(planStorage storage.PlanStorage, planPathProvider execution.PlanPathProvider, diggerExecutor execution.LockingExecutorWrapper) (string, error) {

